### ISSUE TITLE
Logging, rather than printing

### ISSUE THREAD

**Issue by jvc26** [pyroven/django-pyroven#23]
I note at present, the system currently is verbose via ```print``` statements. These should really be fixed to pass to the Django logger.

**Comment by doismellburning** [comment]
Agreed; can we wait on this 'til #27 is in though please, to avoid tedious merge conflicts...

**Comment by jvc26** [comment]
Sounds sensible - will get that merged shortly.
